### ISSUE TITLE
[bug] Hue color selector jumps to a different position

### ISSUE THREAD

**Issue by kennymc-c** [unfoldedcircle/bug-reports#134]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

I noticed a bug when changing the color of a Philips Hue color bulb (LCT007) via the Hue integration. The color selection point jumps to a different position after an adjustment has been made. This issue only happens when I use the Hue integration and does occur when controlling the lamp though HASS. It also does not affect led stripes controlled by a third party and hue compatible led controller (Dresden Elektronik FLS-PP3)

https://github.com/unfoldedcircle/feature-and-bug-tracker/assets/14995555/a7a2f982-cde5-4232-9d23-992b92eb8f8a

### How to Reproduce

1. Open a Hue color lamp on the remote
2. Swipe to the second page
3. Adjust the color selector

### Expected behavior

The color selector should stay at the same position after an adjustment

### System version

1.4.2

### What part of the system affected by the problem?

Integration

### Additional context

Core, UI, Hue & HASS debug logs: [Hue Color Bug.txt](https://github.com/unfoldedcircle/feature-and-bug-tracker/files/13031337/Hue.Color.Bug.txt)

**Comment by kennymc-c** [comment]
This problem still exists in the recent 2.1.0 firmware

**Comment by Albatross7591** [comment]
This problem exists in firmware 2.9.0, but I think it is solved in  firmware 2.9.1

**Comment by henrikwidlund** [comment]
Yes, the jumping issue is solved in 2.9.1. There's still an outstanding issue where the color on the bulb doesn't match the color wheel, it's slightly off. I don't know if it's an issue with the integration or the core or the remote ui

**Comment by kennymc-c** [comment]
I can also confirm this bug has been solved

**Comment by Albatross7591** [comment]
> Yes, the jumping issue is solved in 2.9.1. There's still an outstanding issue where the color on the bulb doesn't match the color wheel, it's slightly off. I don't know if it's an issue with the integration or the core or the remote ui

I also noticed the color difference (I posted a recording of the difference between UC remote HUE integration, openHAB integration and the HUE app on my phone) on the #beta-feedback channel on discord.
